payu/models/model.py

```diff
@@ -339,6 +366,8 @@ def collate(self):
         raise NotImplementedError
 
     def build_model(self):
+        # Don't search user modules for executable paths
+        self.setup_executable_paths(search_module_paths=False)
```

Comment:
I was just adding that there so exec_path values were defined, otherwise it would break the function (because exec paths are no longer set in `Model.init()`.

But if the intention is to remove it, that's great as I don't have to worry about figuring out how to test it